presto-docs/src/main/sphinx/installation/verifier.rst

```diff
@@ -19,9 +19,13 @@ Create a MySQL database with the following table and load it with the queries yo
         test_catalog VARCHAR(256) NOT NULL,
```

Comment:
Can you add a line to the release notes, including the ALTER TABLE statements that users should run to update their database?